electron/legendary/games.ts

```diff
@@ -536,7 +536,7 @@ Categories=Game;
           os = 'Windows'
           break
         case 'darwin':
-          os = 'MacOS'
```

Comment:
Perhaps should be MacOSX then?

Reply:
Apple's official term for their desktop operating system is now macOS, regardless of if it was previously named OS X or Mac OS X.

For example: [this documentation](https://developer.apple.com/documentation/eventkit/ekreminder)

Reply:
Ok, sounds good.